tests/IVIMmodels/unit_tests/simple_test_run_of_algorithm.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import os
+from pathlib import Path
+from src.standardized.ETP_SRI_LinearFitting import ETP_SRI_LinearFitting
+from src.standardized.IAR_LU_biexp import IAR_LU_biexp
+
+## Simple test code... 
+# Used to just do a test run of an algorithm during development
+def test_run(model, **kwargs):
```

Comment:
I think pytest is confused by these because they start with `test_`. Maybe just renaming the 3 here would fix it.

Reply:
I renamed them, but `python -m pytest` still seems to do 81 tests. Or what was it you were looking at?